### ISSUE TITLE
integrate continuously

### ISSUE THREAD

**Issue by rshorey** [newsdev/nyt-campfin#74]
make travis work so it is clear when we break the tests even if we don't run them locally.

**Comment by rshorey** [comment]
@jeremyjbowers tomorrow can we have another chat about the relationship between pyfec and campfin now that I understand this all much better? Travis is not able to find the fec header files and I want to talk about a system-independent place to put them.

**Comment by rshorey** [comment]
We've decided to put this process on hold due to the fact that I wrote tests that check pyfec and campfin together, not modularly, which is going to make testing on travis really hard. the tests work locally and caused me to improve the code significantly so still a success!

Travis lessons learned:
- Travis is picky about how it pip installs a module from git. It wants the tarball format as far as I can tell.
- It takes a bit of doing to get travis to install an extension in a postgres db. See .travis.yml for how I got that to work.
